tests/tensor/test_var.py

```diff
@@ -245,8 +246,13 @@ def test__getitem__newaxis(x, indices, new_order):
 
 def test_fixed_shape_variable_basic():
     x = TensorVariable(TensorType("int64", shape=(4,)), None)
-    assert isinstance(x.shape, Constant)
-    assert np.array_equal(x.shape.data, (4,))
+    assert x.type.shape == (4,)
+
+    shape_fn = pytensor.function([x], x.shape)
+    opt_shape = shape_fn.maker.fgraph.outputs[0]
+    assert isinstance(opt_shape.owner.op, DeepCopyOp)
+    assert isinstance(opt_shape.owner.inputs[0], Constant)
```

Comment:
This one is expected since you have compiled the graph, do I understand correctly?

Reply:
Yes